src/validation/director.correspondence.address.config.ts

```diff
@@ -0,0 +1,13 @@
+import { DirectorField } from "../model/director.model";
+import { GenericValidationType } from "../model/validation.model";
+import { correspondenceAddressErrorMessageKey } from "../utils/api.enumerations.keys";
+
+export const CorrespondenceAddressValidation: GenericValidationType = {
+  AddressBlank: {
+    ErrorField: {
+      messageKey: correspondenceAddressErrorMessageKey.CORRESPONDENCE_ADDRESS_BLANK,
+      source: [DirectorField.CORRESPONDENCE_ADDRESS_RADIO],
+      link: "director_registered_address"
```

Comment:
sorry but should this be `DirectorField.CORRESPONDENCE_ADDRESS_RADIO` ?

Reply:
Yes it is, that is the id for the radio button on the page which the error link will navigate to (#href). DirectorField.CORRESPONDENCE_ADDRESS_RADIO is the name (source) and id is director_registered_address